### PR TITLE
Add Android example

### PR DESCRIPTION
**Pull request by PaulParis1**
Add Android example configuration

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_cookbook/track_battery_level.markdown [hunk 1]
```diff
@@ -29,7 +29,30 @@ sensor:
 ```
 {% endraw %}
 
+### Android Devices
+
```
Comment (klaasnicolaas) [inline source/_cookbook/track_battery_level.markdown:33]: Maybe add some extra information about the Home Assistant android app etc. Just how it is on the iOS devices

**Review by frenck — CHANGES_REQUESTED** [review]
@PaulParis1 Thanks for the PR, could you please look into the comment left by klaasnicolaas? Thanks! 👍

File: source/_cookbook/track_battery_level.markdown (hunk 1 above)
Comment (PaulParis1) [inline source/_cookbook/track_battery_level.markdown:33]: On your Android device, once the official [Home Assistant companion app](https://companion.home-assistant.io/)  is  installed and connected to your Home Assistance instance, you will be able to display the battery level in the Frontend by adding a [template sensor](https://www.home-assistant.io/integrations/template) to your configuration yaml file.  You can also use the battery [sensor device class](https://www.home-assistant.io/integrations/sensor/#device-class) to dynamically change the icon with the battery level.

**Review by PaulParis1 — COMMENTED** [review]
Added some lines explaining the proces for Android devices. Mainly copied from the iPhone section.

**Comment by PaulParis1** [comment]
Tried to update the contribution, but not sure what I did... anyway, I
added the requested intro for the android devices. (as a comment I think???)
It's my first attempt to contribute here. Correct anything that you deem
necessary or point the way that i should follow,
Thanks,
Paul

File: source/_cookbook/track_battery_level.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_cookbook/track_battery_level.markdown:33]: I have added your text.

**Comment by PaulParis1** [comment]
Thanks

Op vr 10 jan. 2020 16:09 schreef Klaas Schoute <notifications@github.com>:

**Review by frenck — APPROVED** [review]

**Pull request merged**